### PR TITLE
Add tests for pyoxigraph parse and serialize functions

### PR DESCRIPTION
**Pull request by edmondchuc**

### CODE REVIEW

File: python/tests/test_parse.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+from pyoxigraph import *
+
+
+class TestParse(unittest.TestCase):
+    def test_parse(self):
+        input = io.BytesIO(b'<foo> <p> "1" .')
+        result = list(parse(input, "text/turtle", base_iri="http://example.com/"))
+
+        assert result == [
```
Comment (Tpt) [inline python/tests/test_parse.py:12]: I believe it's better to use `self.assertEqual` insert of `assert` in order to get a better integration with `unittest`.

File: python/tests/test_serialize.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                    Literal("1"),
+                )
+            ],
+            output,
+            "text/turtle",
+        )
+
+        assert (
+            output.getvalue() == b'<http://example.com> <http://example.com/p> "1" .\n'
+        )
```
Comment (Tpt) [inline python/tests/test_serialize.py:24]: Same here

**Review by Tpt — COMMENTED** [review]
Thank you so much for your PR.

I would maybe merge the test_parse and test_serialize files into a test_io file to avoid very small files. But it's a very personal preference so feel free to ignore it.

File: python/tests/test_parse.py (hunk 1 above)
Comment (edmondchuc) [reply python/tests/test_parse.py:12]: Ah yes, thank you. I'm too used to writing pytest tests which uses `assert`, oops.

Comment (edmondchuc) [reply python/tests/test_parse.py]: I've moved the tests into a test_io file as well.

**Review by Tpt — APPROVED** [review]
Thank you!

Indeed pytest is much nicer and convenient. Because of the small number of tests I have been lazy and used unittest in order to avoid extra dependencies but moving to pytest might make a lot of sense in the future.

**Pull request merged**